dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -415,6 +415,19 @@ Path findClassLocation(String className) {
     return javaClasses.build();
   }
 
+  /** Returns a set of {@link JavaClass}es in {@code jarPath}. */
+  static ImmutableSet<JavaClass> listClassesInJar(Path jarPath) throws IOException {
+    return listClassesInJar(jarPath, createClassRepository(ImmutableList.of(jarPath)));
+  }
+
+  /**
+   * Returns a set of {@link JavaClass}es in {@code jarPath} via the class path in {@link
```

Comment:
From this comment I don't understand the difference between this method and the previous one. Can you elaborate and clarify?

Reply:
Added comparison.

Reply:
I agree. Should at least the one that takes a `Repository` become an instance method on `Repository`? Then you don't really need the next one at all.

Reply:
The private one that takes a Repository is to have the common logic between the other two. Because one of them is `static` (no need to be as an instance method), the private one is static.

Reply:
As discussed, the two static methods were not needed, as the caller only needed class names. Updated.